### PR TITLE
RUM-280 Create Exo Player integration module

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

Creates an ExoPlayer integration module

### CODE REVIEW

File: integrations/dd-sdk-android-exoplayer/build.gradle.kts
Comment (0xnm) [inline integrations/dd-sdk-android-exoplayer/build.gradle.kts:1]: seems there is only `build.gradle.kts` file in the newly created `dd-sdk-android-exoplayer` module. Is is expected? There is no code so far.

File: sample/tv/src/main/java/com/datadog/android/tv/sample/PlayerActivity.kt [hunk 1]
```diff
@@ -58,10 +64,17 @@ class PlayerActivity : AppCompatActivity() {
...
-            val mediaItem = MediaItem.fromUri(videoStreams.first().getUrl())
+            val streamIdx = Random.Default.nextInt(videoStreams.size)
+            val videoStream = videoStreams[streamIdx]
+            val mediaItem = MediaItem.Builder()
+                .setUri(videoStream.getUrl())
+                .setMediaId(intentUri)
+                .build()
             runOnUiThread {
                 videoPlayer.setMediaItem(mediaItem)
+                Thread.sleep(100)
```
Comment (0xnm) [inline sample/tv/src/main/java/com/datadog/android/tv/sample/PlayerActivity.kt:75]: why this is needed?

File: integrations/dd-sdk-android-exoplayer/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+android {
+    compileSdk = AndroidConfig.TARGET_SDK
+    buildToolsVersion = AndroidConfig.BUILD_TOOLS_VERSION
+
+    defaultConfig {
+        minSdk = AndroidConfig.MIN_SDK
+        targetSdk = AndroidConfig.TARGET_SDK
+    }
+
+    namespace = "com.datadog.android.tv"
```
Comment (0xnm) [inline integrations/dd-sdk-android-exoplayer/build.gradle.kts:49]: Should it be `com.datadog.android.exoplayer`?

**Review by 0xnm — APPROVED** [review]

**Review by mariusc83 — APPROVED** [review]

**Review by jonathanmos — APPROVED** [review]

File: integrations/dd-sdk-android-exoplayer/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+dependencies {
+    api(project(":dd-sdk-android-core"))
+    api(project(":features:dd-sdk-android-rum"))
+    implementation(libs.kotlin)
+    implementation(libs.bundles.exoplayer)
+
+    testImplementation(project(":tools:unit")) {
+        attributes {
+            attribute(
+                ProductFlavorAttr.of("platform"),
```
[the remark this answers is not in the document]
Comment (0xnm) [reply integrations/dd-sdk-android-exoplayer/build.gradle.kts:86]: I originally introduced this during Java 11 migration, the reasoning is here https://github.com/DataDog/dd-sdk-android/commit/11eea118a3decef9f307f03e96fd9f0cff44d7bb#diff-ad7f7da3d0a7bb16adf117ddcd32ae9705dcb7ffa9a7abba31ad15a7ac49ae48R16 - I had to add one more flavor in `tools:unit` with different code for JVM vs ART runtimes. And to match proper flavor for unit tests I had to add this.

**Review by 0xnm — APPROVED** [review]

File: sample/tv/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -11,6 +11,8 @@
     <uses-feature android:name="android.software.leanback" android:required="true" />
     <uses-feature android:name="android.hardware.touchscreen" android:required="false" />
 
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
```
[the remark this answers is not in the document]
Comment (xgouchet) [reply sample/tv/src/main/AndroidManifest.xml:14]: Because Android Lint is complaining as one of our dependency requires it (WorkManager IIRC)

**Pull request merged**